ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/StringTopkKudafTest.java

```diff
@@ -66,8 +67,8 @@ public void shouldAggregateTopKWithLessThanKValues() {
   public void shouldMergeTopK() {
     KsqlAggregateFunction<Object, Object[]> topkKudaf =
             topKFactory.getProperAggregateFunction(argumentType);
-    String[] array1 = new String[]{"123", "Hello", "paper"};
-    String[] array2 = new String[]{"Hi", "456", "Zzz"};
+    String[] array1 = new String[]{"paper", "Hello", "123"};
+    String[] array2 = new String[]{"Zzz", "Hi", "456"};
 
     assertThat("Invalid results.", topkKudaf.getMerger().apply("key", array1, array2),
```

Comment:
Not really in scope of your PR but this is kind of weird. I think other systems order strings alphabetically.

Reply:
Yer, our's uses reverse order. TopX on strings is a bit weird anyway.  Maybe we need `BottomX` too!